beep-beep/kv.md

```diff
@@ -1,27 +1,675 @@
-The error returned if the request fails.  
+To get access from the PHP code, you should put the corresponding dependency
+using [the Composer](https://getcomposer.org/).
 
-2. `Set(in *kvv1.Request, _ *kvv1.Response)`: The arguments: the first argument is a `Request` with the `Items` to set; return value isn't used and present here only because GO's RPC calling convention.  
-The error returned if request fails.  
+```sh
+$ composer require spiral/roadrunner-kv
+```
 
-3. `MGet(in *kvv1.Request, out *kvv1.Response)`: The arguments: the first argument is a `Request` with `Items` which should contain only keys (server doesn't check other fields); the second argument is `Response` with the `Items`. Every item will have `key` and `value` set, but without timeout (See: `TTL`).  
-The error returned if request fails.  
-   
-4. `MExpire(in *kvv1.Request, _ *kvv1.Response)`: The arguments: the first argument is a `Request` with `Items` which should contain keys and timeouts set; return value isn't used and present here only because GO's RPC calling convention.  
-The error returned if request fails.  
+## Configuration
 
-5. `TTL(in *kvv1.Request, out *kvv1.Response)`: The arguments: the first argument is a `Request` with `Items` which should contain keys; return value will contain keys with their timeouts. 
-   The error returned if request fails.  
+After installing all the required dependencies, you need to configure this
+plugin. To enable it add `kv` section to your configuration:
 
-6. `Delete(in *kvv1.Request, _ *kvv1.Response)`: The arguments: the first argument is a `Request` with `Items` which should contain keys to delete; return value isn't used and present here only because GO's RPC calling convention.  
-   The error returned if request fails.  
\ No newline at end of file
+```yaml
+rpc:
+  listen: tcp://127.0.0.1:6001
+
+kv:
+  example:
+    driver: memory
+```
+
+Please note that to interact with the KV, you will also need the RPC defined
+in `rpc` configuration section. You can read more about the configuration and
+methods of creating the RPC connection on the [documentation page here](/beep-beep/rpc.md).
+
+This configuration initializes this plugin with one storage named "`example`".
+In addition, each storage must have a `driver` that indicates the type of
+connection used by those storages. In total, at the moment, 4 different types of
+drivers are available with their own characteristics and additional settings:
+`boltdb`, `redis`, `memcached` and `memory`.
+
+The `memory` and `boltdb` drivers do not require additional binaries and are
+available immediately, while the rest require separate dependencies. Please see
```

Comment:
I guess instead of saying - `while the rest require separate dependencies`, will be better to say - `while the rest require additional setup`. Because you don't need any dependencies in terms of code or smt else, but need additional setup for the user (server) environment.